stream/storage/impl/src/main/java/org/apache/bookkeeper/stream/storage/impl/metadata/RootRangeStoreImpl.java

```diff
@@ -692,7 +696,10 @@ StatusCode verifyStreamRequest(String nsName, String streamName) {
                 return FutureUtils.value(null);
             } else {
                 try {
-                    return FutureUtils.value(StreamProperties.parseFrom(streamPropBytes));
+                    final StreamProperties p = StreamProperties.parseFrom(streamPropBytes);
+                    logger.info("namespace_id={} stream_id={} storage_container_id={} stream_name={} ",
+                            nsId, p.getStreamId(), p.getStorageContainerId(), p.getStreamName());
+                    return FutureUtils.value(p);
```

Comment:
We should do this only when we create/delete/modify streams. Also some context in the log will be useful. eg "Created Stream" etc.

Reply:
moved logging to create and delete. add result of operation.  since i'm only logging the (ns, id, name) tuple, i didn't add a log to modify for now.

Reply:
Now that we have the log in create/delete, we should remove the log from here. This will create a unnecessary noise in logs

Reply:
oops. that was supposed to get removed. rebased.